src/sai.rs

```diff
@@ -0,0 +1,71 @@
+//! SAI pad configuration
+
+#![allow(non_snake_case)]
+
+/// An SAI signal
+pub trait Signal: private::Sealed {}
+
+/// A tag that indicates a SAI TX bit clock pad
+pub enum TX_BCLK {}
+
+/// A tag that indicates a SAI TX frame sync pad
+pub enum TX_SYNC {}
+
+/// A tag that indicates a SAI RX bit clock pad
+pub enum RX_BCLK {}
+
+/// A tag that indicates a SAI RX frame sync pad
+pub enum RX_SYNC {}
+
+/// A tag that indicates a SAI MCLK pad
+pub enum MCLK2 {}
+
+impl Signal for TX_BCLK {}
+impl Signal for TX_SYNC {}
+impl Signal for RX_BCLK {}
+impl Signal for RX_SYNC {}
+impl Signal for MCLK2 {}
+
+mod private {
+    pub trait Sealed {}
+    impl Sealed for super::TX_BCLK {}
+    impl Sealed for super::TX_SYNC {}
+    impl Sealed for super::RX_BCLK {}
+    impl Sealed for super::RX_SYNC {}
+    impl Sealed for super::MCLK2 {}
+}
+
+pub trait Pin: super::IOMUX {
+    const ALT: u32;
+    const DAISY: super::Daisy;
+    type Module: super::consts::Unsigned;
+    type Signal: Signal;
+}
+
+pub trait TxDataPin: super::IOMUX {
+    const ALT: u32;
+    type Module: super::consts::Unsigned;
+    type Index: super::consts::Unsigned;
+}
+
+pub trait RxDataPin: super::IOMUX {
+    const ALT: u32;
+    type Module: super::consts::Unsigned;
+    type Index: super::consts::Unsigned;
+}
```

Comment:
> I'm guessing that the daisy register applies to both

I don't think so, since I think the daisy registers only apply to inputs so they wouldn't apply to TX.

`RxDataPin` does have daisy registers though, good catch. I think it has one for every pin.